packages/kbn-type-summarizer/src/lib/bazel_cli_config.ts

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import Path from 'path';
+import Fs from 'fs';
+
+import { CliError } from './cli_error';
+import { parseCliFlags } from './cli_flags';
+
+const TYPE_SUMMARIZER_PACKAGES = ['@kbn/type-summarizer', '@kbn/crypto'];
```

Comment:
What was the reason for defining what uses the summarize here, and not on the CLI defined in the BUILD.bazel file?

Reply:
Eh, just more comfortable changing JS

Reply:
I have to admit, it's kind of weird that the package changes based on who is calling it. But I can also understand that updating a bunch of individual files could be a PITA. I am fine keeping this here knowing it will be temporary.